docs/source/templates/llm_ragas.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Evaluate RAG with Ragas
+type: templates
+category: LLM Evaluations
+cat: llm-evaluations
+order: 970
+is_new: t
+meta_description: Use Ragas metrics to evaluation LLM responses. 
+date: 2024-07-26 14:49:57
+---
+
+<img src="/images/templates/evaluate-rag-automated-metrics.png" alt="" class="gif-border" />
+
+This template uses the [Ragas](https://docs.ragas.io/en/stable/) framework to evaluate your RAG pipeline. When given a prompt, it will use Ragas and OpenAI to return the following:
+
+* An LLM-generated response to the prompt (the ML backend example uses OpenAI). 
+* A Ragas score for faithfulness and accuracy.
+* An LLM-generated evaluation of the response. 
+* A comprehensive overview of precisely which documents were used for context. 
+
+## Prerequisites
+
+This template requires an ML backend to work. Follow the instructions outlined in [RAG Quickstart Labeling](https://github.com/HumanSignal/label-studio-ml-backend/tree/master/label_studio_ml/examples/rag_quickstart) to connect the ML backend to your project. 
+
```

Comment:
This link is broken for now - should work once that example is merged into master.